### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by nongdenchet** [nguyenduchoan9/JournalApp#1]
### Good stuffs
- Using `CustomAdapter` with `ListView`
- Good use of `Toast` to display the message
- Would use of `Bundle` and `Intent` to navigate and pass data
- Would user of `onActivityResult` for editting stuff
- Would to see `MaterialBetterSpinner` in production
### Suggestions
- In Java you should not name your activity like `create_schedule` instead you can use `CreateScheduleActivity`
- Seem like I cant pick the deadline for a todo? Have you added?
- Would be nice if you implement one more stuff `The SQLite` function to store todos in the database, you can use `ActiveAndroid` to save bunch of time
- Your features and UIs is awesome but the code is a little bit hard to follow. I do believe that we can work on this

**Comment by nguyenduchoan9** [comment]
Following your suggestion, i has been improved my application mobile. You help me have more knowledge 's SQLite that i ever been used before.

**Comment by nongdenchet** [comment]
Hi have you receive confirmation email, please check your mail box
